api/deploys/v1/deploys.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package clutch.deploys.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/deploys/v1;deploysv1";
+
+import "google/api/annotations.proto";
+import "api/v1/annotations.proto";
+import "google/protobuf/any.proto";
+import "validate/validate.proto";
+import "google/rpc/status.proto";
+
+service DeploysAPI {
+  rpc TriggerManualDeploy(TriggerManualDeployRequest) returns (TriggerManualDeployResponse) {
+    option (google.api.http) = {
+      post : "/v1/deploys/triggerManual"
+      body : "*"
+    };
+    option (clutch.api.v1.action).type = CREATE;
+  }
+}
+
+enum Environment {
+  ENVIRONMENT_UNSPECIFIED = 0;
+  DEVELOPMENT = 1;
+  STAGING = 2;
+  PRODUCTION = 3;
+}
+
+message DeploymentOptions {
+  bool emergency_rollout = 1;
+  bool skip_deploy_checks = 2;
+  string initiating_user = 3;
+}
```

Comment:
for OSS, I wonder if this should follow the pattern of the Project [proto](https://github.com/lyft/clutch/blob/main/api/core/project/v1/project.proto#L17-L18) where we use `map<string, google.protobuf.Value>`.